ci-operator/config/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master.yaml

```diff
@@ -34,6 +34,7 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  commands: TEST_FOCUS='should provide DNS for the cluster' run-tests
+  commands: TEST_FOCUS='DNS' TEST_SKIP='(\[Disabled:.+\]|\[Disruptive\]|\[Skipped\]|\[Slow\]|\[Flaky\]|\[local\]|\[Local\])'
```

Comment:
@smarterclayton Should these skips be necessary?

Reply:
We need at least `[Disabled:.+]` option to skip the wildcard dns test.

Reply:
I thought that problematic tests are skipped internally in the extended test setup code.

Reply:
Maybe I see the problem now.  Only tests who don't match the exclusion list get the suite conformance tags added to them: https://github.com/openshift/origin/blob/master/test/extended/util/test.go#L117

But I think if you're using TEST_FOCUS the internal exclusion list doesn't matter. I still feel like there must be some other way to specify a focus but also use the exclusion list.

Unless @smarterclayton or @stevekuznetsov knows what to do here I say we merge it and revisit.